index.html

```diff
@@ -1,60 +1,64 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <!-- metadata -->
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-
-    <!-- title and favicon -->
-    <title>Lists of Things</title>
-    <link rel="icon" type="image/x-icon" href="assets/alchemy-favicon.png" />
-
-    <!-- js -->
-    <script type="module" src="app.js"></script>
-
-    <!-- fonts -->
-    <!-- see https://fonts.google.com/ -->
-
-    <!-- css -->
-    <link rel="stylesheet" href="styles/reset.css" />
-    <link rel="stylesheet" href="styles/global.css" />
-    <!-- add link for page specific css -->
-
-</head>
-
-<body>
-    <header>
-        <img class="logo" src="assets/html_css_js.png" alt="logo">
-        <h1>Lists of Things</h1>
-    </header>
-
-    <main>
-        <section>
-            <h2>List 1</h2>
-            <ul id="list-1"></ul>
-        </section>
-
-        <section>
-            <h2>List 2</h2>
-            <ul id="list-2"></ul>
-        </section>
-
-        <section>
-            <h2>List 3</h2>
-            <ul id="list-3"></ul>
-        </section>
-
-        <section>
-            <h2>List 4</h2>
-            <ul id="list-4"></ul>
-        </section>
-
-
-
-    </main>
-
-</body>
-
-</html>
\ No newline at end of file
+    <head>
+        <!-- metadata -->
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+
+        <!-- title and favicon -->
+        <title>Lists of Things</title>
+        <link rel="icon" type="image/x-icon" href="assets/alchemy-favicon.png" />
+
+        <!-- js -->
+        <script type="module" src="app.js"></script>
+
+        <!-- fonts -->
+        <!-- see https://fonts.google.com/ -->
+
+        <!-- css -->
+        <link rel="stylesheet" href="styles/reset.css" />
+        <link rel="stylesheet" href="styles/global.css" />
+        <!-- add link for page specific css -->
+    </head>
+
+    <body>
+        <header>
+            <img class="logo" src="assets/html_css_js.png" alt="logo" />
+            <h1>Lists of Things</h1>
+        </header>
+
+        <main>
+            <section>
+                <h2>Plants</h2>
+                <ul id="plants-list">
+                    <li class="plant-item">
+                        <h2>Philodendron</h2>
+                        <p>Care: bright-indirect light and low-med water</p>
+                    </li>
+                </ul>
```

Comment:
good job typing out what you want your element to look like, this makes it easier to figure out what our function needs to look like. next step would be remembering to delete this from HTML so it doesn't render twice